extensions/book/README.md

```diff
@@ -0,0 +1,2 @@
+# Book extension for Azure Data Studio
+Download sample book https://github.com/lucyzhang929/lucyzhang929.github.io.git and open folder in Azure Data Studio to get started.
```

Comment:
>https://github.com/lucyzhang929/lucyzhang929.github.io.git [](start = 21, length = 58)

Best not to have something link to your personal github account. I'd run this README by the PMs to see what we should put here since it's a little light on the details as of now and is missing stuff like the Privacy/Code of Conduct/License sections - see [https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/README.md](https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/README.md) for an example

Reply:
I replaced my github link with the official Jupyter-Book example